docs/DeveloperGuide.md

```diff
@@ -162,6 +162,32 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add Command Implementation
+
+The `AddCommand` implements the abstract class `Command` to add a new `Employee` to the `Model`.
+
+Given below is how the `AddCommand` operates:
+
+Step 1. After the user inputs all the `prefixes` necessary, which are `Name`, `Phone`, `Address`, `Email`, `Team`, `Role`, and optionally `Tag`, the `AddCommandParser` parses the input and creates a new `Employee` for the `AddCommand`.
+
+Step 2. The `AddCommand` will then store the added `Employee` and feed it to the `execute` method.
+
+Step 3. The `execute` method then gives the `Employee` parameter to the `addEmployee` method under the `Model` interface.
+
+Step 4. The `addEmployee` adds the `Employee` to the `Model`.
+
+The following class diagram shows the structure of an `Employee`:
+
+<puml src="diagrams/UpdatedEmployeeClassDiagram.puml" width="450" />
+
+And this is the sequence diagram that describes the steps:
+
+<puml src="diagrams/DeleteSequenceDiagram.puml" width="550" />
```

Comment:
Is this supposed to be the DeleteSequenceDiagram?

Reply:
Replaced with AddSequenceDiagram in new commit